### PR TITLE
Add properties to the EtheremTransaction and EthereumEvent classes

### PR DESCRIPTION
**Pull request by fordN**
Depends on graphprotocol/graph-node/pull/552

The `EthereumEvent` class been updated to reflect the changes made to [Graph node](https://github.com/graphprotocol/graph-node).

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -668,16 +668,22 @@ export class EthereumBlock {
  */
 export class EthereumTransaction {
   hash: Bytes
-  blockHash: Bytes
-  blockNumber: BigInt
+  index: BigInt
+  from: Address
+  to: Address
```
Comment (Jannis) [inline index.ts]: Wherever we have an optional value in Rust / rust-web3, we should do `T | null` here (with `T` being the type of the value if it is set.

Comment (Jannis) [reply index.ts]: Done (pushed to this PR).

**Comment by fordN** [comment]
Updated to make `logType` optional value as well.

File: index.ts [hunk 2]
```diff
@@ -661,23 +661,30 @@ export class EthereumBlock {
...
 }
 
 /**
  * Common representation for Ethereum smart contract events.
  */
 export class EthereumEvent {
   address: Address
+  logIndex: BigInt
+  transactionLogIndex: BigInt
+  logType: String | null
```
Comment (Jannis) [inline index.ts]: We're using `string` everywhere else and I believe there are some subtle differences between `String` and `string` in AssemblyScript. So let's change this to `string`.

**Pull request merged**